x/bank/handler.go

```diff
@@ -30,8 +30,10 @@ func NewHandler(k keeper.Keeper) sdk.Handler {
 
 // Handle MsgSend.
 func handleMsgSend(ctx sdk.Context, k keeper.Keeper, msg *types.MsgSend) (*sdk.Result, error) {
-	if !k.GetSendEnabled(ctx) {
-		return nil, types.ErrSendDisabled
+	for _, a := range msg.Amount {
+		if !k.GetSendEnabled(ctx, a.Denom) {
+			return nil, sdkerrors.Wrapf(types.ErrSendDisabled, "%s transfers are currently disabled", a.Denom)
+		}
```

Comment:
can we create a util function for this part?

```go
func CoinsSendEnabled(ctx sdk.Context, coins ...sdk.Coin) error {
	for _, coin := range coins {
		if !k.GetSendEnabled(ctx, coin.Denom) {
			return sdkerrors.Wrapf(types.ErrSendDisabled, "%s transfers are currently disabled", coin.Denom)
		}
	}
	return nil
}
```

Reply:
Utility function created with test coverage.
https://github.com/cosmos/cosmos-sdk/blob/9308435ca2e6f7fd3230edb76cb1d4c3e203a682/x/bank/keeper/send.go#L276-L286